Source/Scene/GroundPrimitive.js

```diff
@@ -425,7 +425,7 @@ define([
 
     function getComputeMinimumHeightFunction(primitive) {
         return function(granularity, ellipsoid) {
-            return primitive._minHeight;
+            return primitive._minHeight - 15;
```

Comment:
View parameter - where is the camera and what is the FOV.  This extrusion should happen in the vertex shader on a frame by frame basis so we don't create huge volumes when zoomed that create a high fillrate load.